pai/__main__.py

```diff
@@ -0,0 +1,57 @@
+import argparse
+from pandasai import PandasAI
+from pandasai.llm.openai import OpenAI
+from pandasai.llm.open_assistant import OpenAssistant
+import pandas as pd
+
+import os
+from dotenv import load_dotenv
+
+dotenv_path = os.path.join(os.path.dirname(__file__), '..', '.env')
+load_dotenv(dotenv_path=dotenv_path)
+
+parser = argparse.ArgumentParser(description='Pandas AI command-line tool.')
```

Comment:
in the case of cli we could use the [click package](https://click.palletsprojects.com/en/8.1.x/) to simplify the implementation and maintenance - the code is more organized